app/assets/stylesheets/petitions/admin/_actions.scss

```diff
@@ -0,0 +1,17 @@
+nav.actions {
```

Comment:
Nitpick: Don't need the `nav` node (or `li` below) unless we are specifically upping specificty